cmake/features.cmake

```diff
@@ -74,6 +75,10 @@ if (NOT THREADING STREQUAL "TBB" AND
     NOT THREADING STREQUAL "SEQ")
     message(FATAL_ERROR "THREADING should be set to TBB, TBB_AUTO, OMP or SEQ. Default option is ${THREADING_DEFAULT}")
 endif()
+if(SELECTIVE_BUILD STREQUAL "ON" OR SELECTIVE_BUILD STREQUAL "SELECT")
+    set(THREADING "SEQ")
+    message("-- Disable TBB due to SELECTIVE_BUILD is ${SELECTIVE_BUILD}")
```

Comment:
I suppose we need to turn TBB BIND only off only iff during `COLLECT` we have not used TBB BIND (no NUMA, no HYBRID cores).
BTW, the mode SELECT is not available, only COLLECT :)

Why do you want to turn TBB off at all? I suppose that unused CPU primitives, optimized with TBB, are automatically optimized out if they are not used during inference.

Reply:
Yes, I agree with you, this PR is an immature solution.